cilium/cmd/kvstore_delete.go

```diff
@@ -36,7 +36,7 @@ var kvstoreDeleteCmd = &cobra.Command{
 				Fatalf("Unable to delete keys: %s", err)
```

Comment:
Not related to this spot, but this backport is missing deleting all the package global functions in `pkg/kvstore/kvstore.go` (https://github.com/cilium/cilium/commit/76c7fcad95a651f22128af96aa8d546682da3cd9#diff-fb5855a8577b97a3f31a2f1fe6c75009L49-L169)